lib/vsc/utils/script_tools.py

```diff
@@ -70,6 +71,7 @@ def _script_name(full_name):
 
 DEFAULT_OPTIONS = {
     'disable-locking': ('do NOT protect this script by a file-based lock', None, 'store_true', False),
+    'disable-nagios': ('do NOT generate a nagios cache file. Implies no locking.', None, 'store_true', False),
```

Comment:
why does it imly no locking. that sounds pretty dangerous

Reply:
yeah, that's wrong, need to fix the comment.